master/buildbot/www/service.py

```diff
@@ -132,6 +133,12 @@ def uid(self):
         return jwt.encode(claims, self.site.session_secret, algorithm=SESSION_SECRET_ALGORITHM)
 
 
+class BuildbotLogFile(LogFile):
```

Comment:
The stack trace lead me to https://github.com/twisted/twisted/blob/trunk/src/twisted/web/http.py#L2801

which had all the code to do the decoding. just neded to set self._nativeize